.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -92,7 +92,9 @@ jobs:
 
       - name: Run tests
         timeout-minutes: 100
-        run: bash tests/integration/kubernetes/gha-run.sh run-tests
+        shell: 'script -q -e -c "bash --noprofile --norc -eo pipefail {0}"'
```

Comment:
Could we add (here and below) a one-liner comment to explain why this is needed?

Reply:
Done.